libcodechecker/util.py

```diff
@@ -373,3 +373,17 @@ def get_line(file_name, line_no):
             return ''
     except IOError:
         return ''
+
+
+def get_user_input(msg):
+    """
+    Get the user input.
+
+    :returns: True/False based on the asnwer from the user
+    """
+    yes = ['y', 'yes']
+
+    answer = raw_input(msg)
+    if answer.lower() in yes:
+        return True
```

Comment:
Pure boolean return with condition can be simplified. Actually, this whole method can be a lot more simplified:

```py
def m(msg):
  return raw_input(msg).lower() in ['y', 'yes']
```